packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -209,6 +214,9 @@ class Auth {
         this.fetchedToken = config.fetchedToken || ''
 
         this.OCAPISessionsURL = config.OCAPISessionsURL || ''
+
+        this.clientSecret = config.clientSecret || ''
+        this.silenceWarnings = config.silenceWarnings || false
```

Comment:
We probably will want to include some note in the readme or a doc somewhere explaining how to enable this warning suppression for prod environments.

Reply:
Thanks for the feedback. I've added a note about it in the readme